### DISCUSSION TITLE
test case usage

### DISCUSSION THREAD

**Discussion by kirachid** [ibpsa/project1-boptest#682] [Q&A - Test Cases] [answered]
**Dear
First of all, I would like to thank you, I am Kidari, a PhD student in confort thermal in building at Moulay Ismail University in Morocco. 

I am interested in the BOPTEST/project1 project especially bestest_hydronic_heat_pump I would like to ask you for some help to understand it more as well as to get Testid to activate the code.
Thanks

Note that I have reached the stage shown in the image**

![Capture d’écran 2024-09-22 135920](https://github.com/user-attachments/assets/d3f19a30-ec42-4b96-ac68-8d649a4d70c1)

**Comment by EttoreZ** [comment]
Hey Rachid, here is a colab example that will guide you on how to interact with the online version of Boptest https://colab.research.google.com/github/ibpsa/project1-boptest/blob/master/docs/workshops/RlemWorkshop_20231112/Introduction_to_the_BOPTEST_framework.ipynb.

**Comment by kirachid** [comment]
*Good evening Ettore,Thank you for your reaction to my message.I have
already run this colab until " env.stop() ".I would like to ask you to
guide me to the next step.*
*Note that I have reached the stage shown in the image*

*Rachid Kidari *
*Doctorant  *


*Département de physique Faculté des Sciences et Techniques Errachidia *
*BP 509 Boutalamine 52000 Errachidia/ Maroc*

*Email : r.kidari @edu.umi.ac.ma ***@***.***> Mobile :
+212673975794*

**Reply by kirachid** [reply]
![Uploading Capture d’écran 2024-09-23 003751.png…]()

**Comment by EttoreZ** [comment]
I see, it seems to be an issue with the BOPTESTgym interface maybe, @javiarrobas any thoughts?

**Reply by kirachid** [reply]
ok Ettore 
waiting for you, thanks

**Comment by kirachid** [comment]
I ask you to guide me through the next steps of this BOPTEST project.
Because I like this project and would like to research it if it's okay with you

**Comment by javiarrobas** [comment, marked as answer]
@kirachid thanks for reaching out, I'm happy to see that you're interested in the project!
I suggest you visit our [training page](https://ibpsa.github.io/project1-boptest/training/index.html).
If you want to join our online meetings and get familiar with the next steps, please join our Google Group [here](https://groups.google.com/g/ibpsa-boptest).

**Reply by kirachid** [reply]
I seem to have an issue with the BOPTESTgym interface

**Reply by kirachid** [reply]
![Capture d’écran 2024-09-25 154301](https://github.com/user-attachments/assets/7f65b5bc-3a43-4689-8234-82ab2fbef57c)

**Reply by kirachid** [reply]
![Capture d’écran 2024-09-25 154543](https://github.com/user-attachments/assets/ddb56136-23cd-436f-99b8-1ba07585ae15)
